.github/workflows/pythonpackage.yml

```diff
@@ -95,17 +93,17 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest]
-        python-version: [3.6, 3.7]
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Update Python ${{ matrix.python-version }} tools
-      run: python -m pip install --upgrade pip setuptools wheel
+    # actions/setup-python@v1 cannot be used on macos (see #386)
     - name: Install dependencies for macOS
       run: |
+        pip3 install virtualenv
+        virtualenv venv
+        VIRTUAL_ENV=`pwd`/venv
+        echo "::set-env name=VIRTUAL_ENV::$VIRTUAL_ENV"
+        echo "::add-path::$VIRTUAL_ENV/bin"
```

Comment:
Are these a workaround for a problem sourcing the virtualenv?

Reply:
Yes. Otherwise, you have to source the virtualenv every step like I did at f5b0b5a.
And you would need a virtualenv when you use the system python (the one installed by default), otherwise you will get a permission error, which you can avoid by adding `--user` option to every pip command.